src/routes/transactions/mappers/creation-transaction/creation-transaction.mapper.ts

```diff
@@ -19,27 +19,25 @@ export class CreationTransactionMapper {
       chainId,
       transaction.creator,
     );
-    const transactionHash = transaction.transactionHash;
     const implementation = transaction.masterCopy
       ? await this.addressInfoHelper.getOrDefault(
           chainId,
           transaction.masterCopy,
         )
       : null;
-    const factory = transaction.factoryAddress
-      ? await this.addressInfoHelper.getOrDefault(
-          chainId,
-          transaction.factoryAddress,
-        )
-      : null;
+    const factory = await this.addressInfoHelper.getOrDefault(
+      chainId,
+      transaction.factoryAddress,
+    );
     const txInfo = new CreationTransactionInfo(
       creator,
-      transactionHash,
+      transaction.transactionHash,
       implementation,
       factory,
     );
+    const type = 'creation';
```

Comment:
This can be static.

Reply:
What you mean?

Reply:
Right now this constant is part of the function stack. There's no need for it. We use it as global constant (static)